examples/experiment-1.0.amp.html

```diff
@@ -53,11 +81,25 @@ <h1>Title TBD</h1>
           "variants": {
             "solid": {
               "weight": 50,
-              "mutations": [{}]
+              "mutations": [
+                {
+                  "type": "attributes",
+                  "target": "h1",
+                  "attributeName": "style",
+                  "value": "border: 1px black solid"
+                }
+              ]
             },
             "dashed": {
               "weight": 50,
-              "mutations": [{}]
+              "mutations": [
+                {
+                  "type": "attributes",
+                  "target": "h1",
+                  "attributeName": "style",
```

Comment:
Have we settled down on the supported attributeName list? I'm also a bit worried that one can change the attribute value to something that's not valid.

Reply:
@zhouyx Oooh, so parsing the actual "AttributeMutation" that will be in another PR. But you make a good point. I'll go ahead and leave a todo. I think for the initial experiment, we will only support setting the style attribute. 

cc @jeffjose

Reply:
I see. Please leave a TODO.  I know there's restriction to the value of style attribute as well. 
For example 
```
<body style="position: fixed">Hello, AMP world.</body>
```
is invalid. 

What's our plan to validate attribute value?

Reply:
Discussed offline, and added a TODO in `mutation-parser.js`.